internal/cocoasdk/api_coregraphics_darwin.go

```diff
@@ -0,0 +1,5 @@
+package cocoasdk
```

Comment:
I'm not sure, but what about just `cocoa`? Do you have a plan to use other similar packages? I'd like to know your plan.

Reply:
Cocoa is really just a collection of most of the graphical frameworks like AppKit, Foundation etc. I think just cocoa is fine as the name. The goal is to port enough to make all of ebitengine including glfw into purego.

Reply:
I mean I wanted to know you would add other packages besides `cocoa` and I wanted to know the naming convention. Anway we can revisit this later. Let's go with `cocoa`.